test/evergreen.yml

```diff
@@ -1864,6 +1864,8 @@ tasks:
     - name: compile
     - name: generate-datafile-little-endian
       variant: little-endian
+    - name: verify-datafile-little-endian
```

Comment:
These are to avoid needless work when the `generate-datafile-XXX-endian` task fails. 
If a datafile generated on little-endian cannot be verified on little-endian there is no reason to also check on a big-endian platform.